app/views/users/show.html.erb

```diff
@@ -15,12 +15,28 @@
     <section>
       <h3>Followers</h3>
     <% @followers.each do |resp| %>
-      <%= link_to resp.login, resp.html_url %><br/>
+      <section class = "follower-<%= resp.login %>">
+      <%= link_to resp.login, resp.html_url %>
+      <% if User.github_usernames.include?(resp.login) %>
+      <%= link_to "Add as Friend", user_friends_path(current_user.id, params: {:github_username => resp.login}), method: :post %><br/>
```

Comment:
could probably stand to take the hash rockets out (i.e. {github_username: resp.login})